buildroot-external/board/generic-aarch64/grub.cfg

```
@@ -0,0 +1,58 @@
+set default=99
+set timeout=5
+
+set ORDER="A B"
+set A_OK=0
+set B_OK=0
+set A_TRY=0
+set B_TRY=0
+set MACHINE_ID=""
+load_env
+
+# select bootable slot
+for SLOT in $ORDER; do
+    if [ "$SLOT" == "A" ]; then
+        INDEX=0
+        OK=$A_OK
+        TRY=$A_TRY
+        A_TRY=1
+    fi
+    if [ "$SLOT" == "B" ]; then
+        INDEX=1
+        OK=$B_OK
+        TRY=$B_TRY
+        B_TRY=1
+    fi
+    if [ "$OK" -eq 1 -a "$TRY" -eq 0 ]; then
+        default=$INDEX
+        break
+    fi
+done
+
+# reset booted flags
+if [ "$default" -eq 99 ]; then
+    if [ "$A_OK" -eq 1 -a "$A_TRY" -eq 1 ]; then
+        A_TRY=0
+    fi
+    if [ "$B_OK" -eq 1 -a "$B_TRY" -eq 1 ]; then
+        B_TRY=0
+    fi
+    default=0
+fi
+
+save_env A_TRY A_OK B_TRY B_OK ORDER MACHINE_ID
+
+source ($root)/cmdline.txt
```

I then would add the default args here:

```suggestion
default_cmdline="rootwait zram.enabled=1 zram.num_devices=3 apparmor=1 security=apparmor systemd.machine_id=$MACHINE_ID cgroup_enable=memory fsck.repair=yes"
source ($root)/cmdline.txt
```